package.json

```diff
@@ -24,9 +25,11 @@
     "eslint": "^6.8.0",
     "mocha": "^7.0.1",
     "proxyquire": "^2.1.3",
+    "should": "^13.2.3",
```

Comment:
We use `should` [module](https://www.npmjs.com/package/should), so we can use it instead of `chai.should()`. Otherwise we can get rid of it.